docs/source/quick_start/introduction.rst

```diff
@@ -0,0 +1,40 @@
+
+What is MCDR
+============
+
+MCDReforged (MCDR, hereinafter) is a tool to manage your Minecraft server with the custom plugin system. No need to modify the original Minecraft server at all
+
+In-game calculator, player high-light, scoreboard manipulate, structure file management, backup / rollback... you can control the whole world with MCDR and its plugins
+
+Greatly thanks to `chino_desu <https://github.com/kafuuchino-desu>`__ and his `MCDaemon 1.0 <https://github.com/kafuuchino-desu/MCDaemon>`__ for the idea of such a cool tool
+
+Advantages
+----------
+
+Vanilla
+~~~~~~~
+
+MCDR runs above the server. It doesn't need to modify the server at all, which keep everything **vanilla**
+
+Plugin System
+~~~~~~~~~~~~~
+
+MCDR has a hot-reloadable plugin system. You don't need to shut down the server to update the plugins
```

Comment:
"hot-reloadable plugin system" itself is not an advantage. It should be "A rich ecosystem of plugins from the community". "hot-reloadable plugin system" is useless if there's no plugin to hot-reload

Reply:
that from your README though

Reply:
I think you were trying to compare itwitho Minecraft mods / plugins

Reply:
fair. just keep it as-is, I'll change them later